app/views/mail_shared/panel/_row_content_image.html.haml

```diff
@@ -0,0 +1,17 @@
+- last == true ? last_class = "last" : last_class = ""
```

Comment:
ok cool - I am not super familiar with INK but I think I remember this being related to that? it's for the last row in a table? or last td in a row?